wdl/src/bin/wdl.rs

```diff
@@ -433,6 +450,9 @@ enum Command {
 
     /// Formats a WDL file.
     Format(FormatCommand),
+
+    /// Documents a workspace.
+    Document(DocumentCommand),
```

Comment:
Should this be just `Doc` to correspond with the package name (`wdl-doc` and `wdl doc`)?

Reply:
I don't think it really matters either way, as this is really only ever meant for dev testing. No end user should be running `wdl doc/document`, so sure I can change it 🤷